app/views/layouts/application.html.erb

```diff
@@ -6,9 +6,12 @@
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
need to require bootstrap as rails gem https://github.com/twbs/bootstrap-rubygem